src/screens/ProfileScreen/index.js

```diff
@@ -1,6 +1,8 @@
+import netInfo from '@react-native-community/netinfo';
+import {useNavigation} from '@react-navigation/core';
+import {debounce} from 'lodash';
+import PropTypes from 'prop-types';
 import * as React from 'react';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed and some imports have been removed. Ensure that these changes do not break any dependencies in the code.
```diff
- import * as React from 'react';
- import ImagePicker from 'react-native-image-crop-picker';
- import Toast from 'react-native-simple-toast';
+ import netInfo from '@react-native-community/netinfo';
+ import {useNavigation} from '@react-navigation/core';
+ import {debounce} from 'lodash';
+ import PropTypes from 'prop-types';
+ import * as React from 'react';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->